doc/sphinxext/announce.py

```diff
@@ -68,8 +68,21 @@ def get_authors(revision_range):
         revision_range = f"{lst_release}..{cur_release}"
 
     # authors, in current release and previous to current release.
-    cur = set(re.findall(pat, this_repo.git.shortlog("-s", revision_range), re.M))
-    pre = set(re.findall(pat, this_repo.git.shortlog("-s", lst_release), re.M))
+    # We need two passes over the log for cur and prev, one to get the
+    # "Co-authored by" commits, which come from backports by the bot,
+    # and one for regular commits.
+    xpr = re.compile(r"Co-authored-by: (?P<name>[^<]+) ")
```

Comment:
I might be responsible for some issues here; I typically clear out all commit messages before squashing

Reply:
Yeah, we'll want to keep it here *for backports* (GitHub also uses the message I think).

Shouldn't matter for regular PRs, since they use the regular author field, not the message.